CommonUI/src/main/res/values/dimens.xml

```diff
@@ -17,6 +17,8 @@
     <dimen name="dk_ic_small">18dp</dimen>
     <dimen name="dk_ic_medium">26dp</dimen>
     <dimen name="dk_ic_big">34dp</dimen>
+    <dimen name="dk_ic_xbig">46dp</dimen>
```

Comment:
@mohamed-garci Are these 2 values validated by the product team ? Is it possible to have a multiple of 8 ? (i.e 48dp instead of 46)

Reply:
@Steven-DriveQuant these 2 values are not validated by the product team.
I'll add the correct values in the dedicated file, and validate with the product team.